personal/privacyidea/classic/mfatype_registration/phase02.tpl

```diff
@@ -0,0 +1,2 @@
+<h1>THIS FILE SHOULD NEVER BE VISIBLE / LOADED.</h1>
+<h1>THIS PHASE IS RESERVED FOR THE PDF DOWNLOAD.</h1>
```

Comment:
I figured it would be confusing to *not* have them, since they are actually 3 Phases for the registration token now.  But yes, they are loaded but never displayed since ob_end_clean() gets called.

I would say the easiest way would be to make `preparePhase` and `evaluatePhase` return `null|bool` bool indicates success and null indicates that no template need to be loaded.